osu.Game.Rulesets.Taiko/Difficulty/TaikoPerformanceCalculator.cs

```diff
@@ -64,21 +63,17 @@ private double computeDifficultyValue()
         {
             double difficultyValue = Math.Pow(5.0 * Math.Max(1.0, Attributes.StarRating / 0.0075) - 4.0, 2.0) / 100000.0;
 
-            // Longer maps are worth more
             double lengthBonus = 1 + 0.1 * Math.Min(1.0, totalHits / 1500.0);
             difficultyValue *= lengthBonus;
 
-            // Penalize misses exponentially. This mainly fixes tag4 maps and the likes until a per-hitobject solution is available
```

Comment:
Not sure about the removal of comments like this. Are they no longer applicable, or to be removed soon, or just inscrutable? Asking because there's some info here that cannot be conveyed by code and I'm worried that potentially important context will be lost - unless the comments are already illegible enough to be useless.

Reply:
This comment in particular makes zero sense - tag4 maps are completely irrelevant in taiko. A lot of comments were copied from osu perfcalc without changing or even considering usefulness, for example when acc pp values being straight up scaled by accuracy and comment saying that its related to normal distribution (which it is in osu ruleset).

Reply:
I see, thanks for explaining.